### ISSUE TITLE
Divvun keyboards missing in Play store

### ISSUE THREAD

**Issue by trondtynnol** [divvun/divvun-keyboard#11]
I and Børre just tried installing Divvun keyboards on a Swedish user’s Android phone, but could not find the app, only Divvun Dev keyboards. This seemed to be a quite new phone. 

May there be some problems with the distribution in Sweden?

**Comment by snomos** [comment]
No, there should not be. @SteffenErn could you prepare a new release of the Android keyboard app?
